protocol/app/ante/sigverify.go

```diff
@@ -63,7 +64,10 @@ func (svd SigVerificationDecorator) AnteHandle(
 		return ctx, err
 	}
 
-	// check that signer length and signature length are the same
+	// Check that signer length and signature length are the same.
+	// If the lengths are the same, then sigs and signers are ordered correctly. This means that each expected signer
+	// provided a signature. In cosmos original sigverify, the ordering is explicitly checked.
```

Comment:
> If the lengths are the same, then sigs and signers are ordered correctly.

I don't really follow this comment

>  This means that each expected signer provided a signature. In cosmos original sigverify, the ordering is explicitly checked.

This comment seems to imply that we are making different ordering assumption from in the original implementation (v0.50.x [release](https://github.com/cosmos/cosmos-sdk/blame/release/v0.50.x/x/auth/ante/sigverify.go#L323)), which is not true